.github/workflows/main.yml

```diff
@@ -132,3 +133,22 @@ jobs:
         run: yarn test
         env:
           CI: true
+  test-rust-swc-plugin:
+    name: Build and Unit Test SWC Plugin
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          target: wasm32-wasi
+          override: true
+      - uses: marcopolo/cargo@a527bf4d534717ff4424a84446c5d710f8833139
```

Comment:
maybe, we could use a short SHA here instead of the full one, this's an action that's based on [actions-rs/cargo@v1](https://github.com/actions-rs/cargo), but unfortunately actions-rs/cargo@v1 doesn't support setting working directory in the CI to build nested rust projects, so this guy (marcopolo) made a PR that implements it but it never got merged, so I'm using his action instead which is exactly the same as the official one, but pinned it to a git SHA, so that we don't depend on the guy's `master` branch, as he was proposing here
https://github.com/actions-rs/cargo/issues/6#issuecomment-614256424